MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -863,6 +881,10 @@
          bookmarks tray button then a new item is bookmarked. -->
     <string name="bookmarks_saved_notification">Saved to Bookmarks!</string>
 
+    <!-- This string is displayed in a notification that appears in the tray bar on top of the
+         downloads tray button then a download finishes. -->
+    <string name="download_completed_notification">%1$s Downloaded</string>
```

Comment:
What is `%1$s`? Please always add in the comment what replaces the variables.

Reply:
Fixed